src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -3121,6 +3183,15 @@ public sealed partial class X509Store : System.IDisposable
         public void Remove(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate) { }
         public void RemoveRange(System.Security.Cryptography.X509Certificates.X509Certificate2Collection certificates) { }
     }
+    public partial class X509SubjectAlternativeNameExtension : System.Security.Cryptography.X509Certificates.X509Extension
+    {
+        public X509SubjectAlternativeNameExtension() { }
+        public X509SubjectAlternativeNameExtension(byte[] rawData, bool critical = false) { }
+        public X509SubjectAlternativeNameExtension(System.ReadOnlySpan<byte> rawData, bool critical = false) { }
+        public override void CopyFrom(System.Security.Cryptography.AsnEncodedData asnEncodedData) { }
+        public System.Collections.Generic.IEnumerable<string> EnumerateDnsNames() { throw null; }
```

Comment:
Is ordering important here? That is, if the raw data contains interleaved DNS names and IP addresses, is it ok for the caller to perform two enumerations and potentially lose the interleaved ordering?

Reply:
I can't think of a scenario where the interleaving matters, they're on different planes under RFC 6125-style matching.  For someone who has such a scenario, there's AsnReader (also the answer if they want to see a SAN URI/UPN/X500Name/etc).